nodejs/week2/homework/meal-sharing/src/backend/api/reservations-router.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const router = express.Router();
+const reservations = require("./../data/reservations.json");
+router.get("/", async(request, response) => {
+    try {
+        response.json(reservations);
+        console.log("in/api/reservations");
+    } catch (error) {
+        throw error;
+    }
+});
+//Respond with the json for the reservation with the corresponding id 
+router.get("/:id", async(req, res) => {
+    try {
+        const { id } = req.params;
+        const reservationId = reservations.filter((reservation) => reservation.id === parseInt(id));
+        if (reservationId < 1) {
```

Comment:
Operator '<' cannot be applied to types '{ id: number; name: string; email: string; mealId: number; }[]' and 'number'.ts(2365)